app/views/dashboards/show.html.erb

```diff
@@ -1,6 +1,5 @@
 <% title "Dashboard" %>
 
-<main id="main-content">
   <header class="crayons-layout">
```

Comment:
Hey, I was going through this PR to understand some of the frontend codebase. If you look at the failed tests in Node 0 of the Travis Build, test `/spec/system/dashboards/user_visits_dashboard_spec.rb:12` fails with following error - 
``` ruby
Failure/Error:
 within "main > header" do
   expect(page).to have_text("1")
 end
Capybara::ElementNotFound:
 Unable to find css "main > header"
```
That is because the `<main>` tag was shifted and it is no longer a parent of the `<header>` tag. The test should be modified so that it just looks for `header` and not `main > header`. Semantically it makes sense because the analytics partial in the header is not the main content of the page, however analytics gets rendered in this page alone. I am confused about whether it should or should not be considered as main content.

Reply:
@squarebat Hi, Thanks for pointing that out i handled it. It also seem to me that when analytic is chosen, only analytic page is rendered without any other part from show.html.erb so that shouldn't affect analytic page. I am not familiar webpack and ruby so i might be wrong.